src/lib/middleware.ts

```diff
@@ -126,29 +36,6 @@ const setCookieHeader = (id: string, res: ServerResponse): void => {
   res.setHeader("Set-Cookie", serialize(formSubmissionCookieId, id, options));
 };
 
-export async function updateFormCache(
-  submissionId: string,
-  formData: FormSubmission,
-  cache: IFormCache = FormCacheFactory.getCache()
-): Promise<void> {
-  await cache.update(submissionId, formData);
-}
-
-export async function setFormCache(
-  submissionId: string,
-  registration: Registration,
-  cache: IFormCache = FormCacheFactory.getCache()
-): Promise<void> {
-  await cache.set(submissionId, registration);
-}
-
-export async function clearFormCache(
-  submissionId: string,
-  cache: IFormCache = FormCacheFactory.getCache()
-): Promise<void> {
-  await cache.clear(submissionId);
-}
-
```

Comment:
Deleted in favour of using the gateway pattern via `IAppContainer` as previously discussed.